### PR TITLE
UI Control: Navigate to external URL

### LINKED ISSUE #987: HowTo: Use button from dashboard 2.0 to get an external Link.
### Description

Hello,

how can i setup NodeRed to add an exernal Link with an Dashboard 2.0 button.
I found the solution to setup a template genaerate an html button, but i want to use the dashboard 2.0 button.

Thank you very much
Martin

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

- Add support for `msg.payload.url` to trigger navigation to external resources
- Add in-Editor documentation
- Add online documentation
- Adds E2E test for coverage of the `url` option

## Related Issue(s)

Closes #987

### CODE REVIEW

**Comment by joepavitt** [comment]
Test problems may be linked to https://docs.cypress.io/guides/guides/web-security#Different-superdomain-per-test-requires-cyorigin-command (thanks for sharing @cstns)

**Comment by cstns** [comment]
There is a way to circumvent this, it's not the prettiest thing in the world but it works:

```js
cy.click() // get's redirected

cy.origin('https://www.google.com', () => {
    cy.url().should('eq', 'https://www.google.com/search?q=rick+astley')
})
```
You have to pass the new domain to origin. In my case I knew I would be redirected to `https://www.google.com/search?q=rick+astley` so i just manually added the origin

**Comment by joepavitt** [comment]
@cstns this is good for review now

File: ui/src/widgets/ui-control/UIControl.vue
Comment (cstns) [inline ui/src/widgets/ui-control/UIControl.vue:36]: Is this intentional or a leftover?

Comment (joepavitt) [inline ui/src/widgets/ui-control/UIControl.vue:36]:
```suggestion
```

File: ui/src/widgets/ui-control/UIControl.vue
Comment (joepavitt) [inline ui/src/widgets/ui-control/UIControl.vue]:
```suggestion
```

**Comment by joepavitt** [comment]
console logs left in by mistake @cstns - tried removing them via the web interface and it's bugged out, will remove locally when Ig et the chance, locked in something else atm

**Review by cstns — APPROVED** [review]
approving ahead of time, pending web interface bugging out. Once sorted merge at will

**Pull request merged**
